src/regression_tests/regression_test.test.ts

```diff
@@ -0,0 +1,137 @@
+import fetchMock from 'jest-fetch-mock';
+import {
+  createUser,
+  getBalance,
+  getEnvVariables,
+  getPaymailDomain,
+  getTransactions,
+  RegressionTestConfig,
+  RegressionTestUser,
+  removeRegisteredPaymail,
+  sendFunds,
+} from './utils';
+
+const MINIMAL_FUNDS_PER_TRANSACTION = 2;
+const TEST_TIMEOUT_MS = 120000;
```

Comment:
Could you add a comment on how long it is ? I mean ex. FIVE_MINUTES_TIMEOUT

Reply:
Sure, on it